packages/bee-q/src/components/notification/bq-notification.tsx

```diff
@@ -0,0 +1,169 @@
+
+  // Listeners
+  // ==============
+
+  // Public methods API
+  // These methods are exposed on the host element.
+  // Always use two lines.
+  // Public Methods must be async.
+  // Requires JSDocs for public API documentation.
+  // ===============================================
+
+  /**
+   * Trigger function when you want to close Notification
+   */
+  @Method()
+  async hideNotification() {
+    this.closeNotification();
+  }
+
+  /**
+   * Trigger function when you want to show Notification
+   */
+  @Method()
+  async showNotification() {
+    this.setNotificationOpened();
+  }
+
+  // Local methods
+  // Internal business logic.
+  // These methods cannot be called from the host element.
+  // =======================================================
+
+  private getNotificationIconName = () => {
+    if (!this.showIcon) return '';
+
+    const type = this.type;
+    let icon = 'info';
+
+    if (type === 'success') {
+      icon = 'check-circle';
+    }
+
+    if (type === 'error') {
+      icon = 'x-circle';
+    }
+
+    return icon;
+  };
+
+  private handleSlotChange = () => {
+    this.hasBtn = hasSlotContent(this.buttonElem);
+  };
+
+  private closeNotification = () => {
+    this.isHidden = true;
+  };
+
+  private setNotificationOpened = () => {
+    this.isHidden = false;
+  };
+
+  // render() function
+  // Always the last one in the class.
+  // ===================================
+
+  render() {
+    const styles = {
+      ...(this.subjectColor && { color: getColorCSSVariable(this.subjectColor) }),
+    };
+
+    return (
+      <Host aria-hidden={this.isHidden} hidden={this.isHidden}>
+        <div
+          class="bg-white notification-shadow notification-radius inline-block w-auto px-5 py-4"
+          id="notification-main-holder"
+        >
+          {this.showIcon && (
+            <div class="mr-2 inline-block text-left align-top" id="notification-icon-holder">
+              <bq-icon name={this.getNotificationIconName()} color={this.subjectColor} size="18"></bq-icon>
+            </div>
+          )}
+          {!this.showIcon && (
+            <div class="avatar-slot-holder inline-block text-left align-top" id="notification-avatar-holder">
+              <slot name="avatar" />
+            </div>
+          )}
+          <div class="max-w-x inline-block text-left align-top">
+            <div class="title-font font-semibold" style={styles} id="subject">
+              <slot />
+            </div>
+            <div class="description-slot-holder description-font font-regular">
+              <slot name="description" />
+            </div>
+            <div class={{ 'mt-3': this.hasBtn }}>
+              <span class={{ 'mr-2': this.hasBtn }} ref={(spanElem) => (this.buttonElem = spanElem)}>
+                <slot name="first-button" onSlotchange={this.handleSlotChange} />
+              </span>
+              <span>
+                <slot name="second-button" />
+              </span>
+            </div>
```

Comment:
We just need a footer slot here. Is up to the consumer to provide the number of actions needed.
The parent container of the slot should handle spacing between the elements inside, something that you can do with flex and gap 🙂